### PR TITLE
create association module

### PR DESCRIPTION
**Pull request by mrkno**

### CODE REVIEW

File: associate.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
+var associateMap = {};
+
+exports.help = function() {
+	return '/associate "<hook>" "<text>" : Associate and disassociate a phrase with another.';
+};
+
+exports.match = function(text, thread, api) {
+	if (text.startsWith('/associate')) {
```
Comment (Awarua-) [inline associate.js]: Is starts with defined?

File: associate.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+exports.load = function(){};
+
+exports.run = function(api, event) {
+	if (!event.body.startsWith('/associate')) {
+		exports.match(event.body, event.thread_id, api);
+		return;
+	}
+	
+	var spl = event.body.split('"');
+	if (spl.length !== 3 && spl.length !== 5)  {
```
Comment (Awarua-) [inline associate.js]: Not sure what is going on here

File: associate.js (hunk 1 above)
Comment (mrkno) [inline associate.js]: I'm going to add them in server.js cause the number of times they will be useful is high

File: associate.js (hunk 2 above)
Comment (mrkno) [inline associate.js]: Splitting at each " char

**Pull request merged**